docs/collecting-data/collecting-from-own-applications/mobile-trackers/migration-guide/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Migration guide"
+date: "2022-08-30"
+sidebar_position: 1000
```

Comment:
This way there is a weird sequence from previous versions, to Google Play, to this. Can we adjust a bit? Maybe put Google Play stuff before previous versions.

Reply:
Moved before previous versions